test/services/constructions.test.js

```diff
@@ -1,10 +1,82 @@
+/* eslint-disable no-undef */
+
 const assert = require('assert');
+
 const app = require('../../src/app');
+const { clearAll, makeClient } = require('../utils');
+
+const host = app.get('host');
+const port = app.get('port');
+
+describe.only('\'constructions\' service', () => {
+    const users = app.service('users');
+    const constructions = app.service('constructions');
+
+    let server;
+    let client;
+    let service;
+
+    before(async () => {
+        server = app.listen(port);
+
+        await clearAll(users, constructions);
+
+        await users.create({
+            username: 'gauss',
+            password: 'secret',
+            permissions: ['admin'],
+        });
+
+        ({ client, _ } = await makeClient(host, port, 'gauss', 'secret'));
+
+        service = client.service('constructions');
+    });
+
+    after(() => {
+        server.close();
+    });
+
+    describe('user can manage constructions', async () => {
```

Comment:
Test naming style: this test case with underlying `it`s will be reported by the test runner as:

> user can manage constructions
>  - created constructions are assigned to their creator

Instead, please name the tests so that the descriptions match the `describe` and `it` parts: describe something, it will/can/does something, for instance:

```javascript
describe('user', async () => {
    it('creates constructions that are assigned to him', async () => { /* snip */ });
});
```

Which will be reported as:

> user
> - creates constructions assigned to him

which is pretty much informative and gives nice documentation of system behaviour.